db/schema.rb

```diff
@@ -25,6 +25,15 @@
     t.datetime "updated_at"
   end
 
+  create_table "comments", force: :cascade do |t|
```

Comment:
Do you want me to remove the comments table from the schema manually on this branch?